_maps/glowstation.json

```diff
@@ -0,0 +1,18 @@
+{
+	"map_name": "Glow Station",
+	"map_link": "GlowStation",
+	"map_path": "map_files/GlowStation",
+	"map_file": "GlowStation.dmm",
+	"shuttles": {
+		"cargo": "cargo_fland",
+		"ferry": "ferry_fancy",
+		"whiteship": "whiteship_fland",
+		"emergency": "emergency_delta"
+	},
+		"traits": [{"Up" : 1, "Baseturf" : "/turf/open/lava/smooth", "Linkage" : "Cross"}, 
+	{"Up" : 1, "Down" : -1, "Baseturf" : "/turf/open/openspace", "Linkage" : "Cross"},
+	{"Up" : 1, "Down" : -1, "Baseturf" : "/turf/open/openspace", "Linkage" : "Cross"},
+	{"Up" : 1, "Down" : -1, "Baseturf" : "/turf/open/openspace", "Linkage" : "Cross"}, 
+	{"Down" : -1, "Baseturf" : "/turf/open/openspace", "Linkage" : "Cross"}]
```

Comment:
shouldn't the baseturf be different?

Reply:
openspace is the multi-z turf.  its not "SPACE"